test/allowlist_for_publicAPI.json

```diff
@@ -1095,6 +1095,7 @@
     "ComplexFloatStorage",
     "DisableTorchFunction",
     "DisableTorchFunctionSubclass",
+    "EnableTorchFunction",
```

Comment:
Adding new entries to this file is not ok.
You need to either make this function properly public by exposing it well and having it's module correct. Or make it private.

Reply:
so I followed exactly what was done for DisableTorchFunction (which is why I made this public) - when I attempted to change the module I got an error that it was immutable, saw that an exception was made for DisableTorchFunction in the checks, and followed the same pattern for EnableTorchFunction - is there a way to properly set this from the C side?

Reply:
Yeah quite a few older function were added in a broken state before we created this test :/  But we are making all the new ones better!

In this case, you most likely need to fix the `__module__` field on the python side indeed.